bluetooth_mesh/apps/meshcli.py

```diff
@@ -480,8 +480,8 @@ async def __call__(self, application, arguments):
                 )
 
                 resolution = (
-                    self._get_resolution(arguments)
-                    or PublishPeriodStepResolution.RESOLUTION_10_S
+                        self._get_resolution(arguments)
+                        or PublishPeriodStepResolution.RESOLUTION_10_S
                 )
```

Comment:
Are these formatting changes necessary?

Reply:
No, its not, I will revert them. I have some issue with PyCharm formatter which formats code in a different way that it is formatted right now.